src/modules/item/types.ts

```diff
@@ -132,6 +132,8 @@ export type CatalystItem = Omit<BaseItem, 'createdAt' | 'updatedAt'> & {
   collectionAddress: string
 }
 
+export type CatalystTPItem = Omit<CatalystItem, 'rarity' | 'collectionAddress'> & {}
```

Comment:
This is temporary until we have a proper definition of the entity metadata needed